### ISSUE TITLE
Don't understand list-item-spacing

### ISSUE THREAD

**Issue by rugk** [wooorm/remark-lint#73]
I really don't understand [list-item-spacing](https://github.com/wooorm/remark-lint/blob/master/doc/rules.md#list-item-spacing). What the description basically says is just it "corrects looseness".
But what is "looseness" in markdown?

I mean when do you consider a list "loose" and when not? And when should paragraphs be loose and when not? What's the rational behind this rule?

**Comment by wooorm** [comment]
A list item is loose if it's followed by a blank line ('\n\n'), and tight when it's not. This rule wants list items to be loose if at least one of them contains a newline, and is based on this rule: http://www.cirosantilli.com/markdown-style-guide/#empty-lines-inside-lists.

P.S. I suggest using the Gitter channel for more question-like problems! :)

**Comment by Mister-Hope** [comment]
I don't agree with this rule, and it seems this rule can not be disabled by setting the second options to false.

I would just like to create a list of items which are hightlighted:

```md
- **Something A**

- **Something B**

- **Something C**
```

- **Something A**

- **Something B**

- **Something C**

I am leaving blanks to create a `<p>` tag, why should the rule stop this? @wooorm

**Comment by wooorm** [comment]
@Mister-Hope Thanks for reaching out!

Because we treat our issues list as [the team’s backlog](https://en.wikipedia.org/wiki/Scrum_(software_development)#Product_backlog), we close issues that are questions since they don’t represent a task needing to be completed.
See [`support.md`](https://github.com/unifiedjs/.github/blob/main/support.md) for where and how to ask questions.
